pkg/providers/pricing/pricing.go

```diff
@@ -451,3 +393,17 @@ func populateInitialSpotPricing(pricing map[string]float64) map[string]zonal {
 	}
 	return m
 }
+
+func (p *Provider) Reset() {
+	// see if we've got region specific pricing data
+	staticPricing, ok := initialOnDemandPrices[p.region]
+	if !ok {
+		// and if not, fall back to the always available us-east-1
+		staticPricing = initialOnDemandPrices["us-east-1"]
+	}
+
+	p.onDemandPrices = staticPricing
+	p.spotPrices = populateInitialSpotPricing(staticPricing)
+	p.onDemandUpdateTime = initialPriceUpdate
```

Comment:
Is this correct? If it's getting reset, shouldn't the update time be `time.Now()`?

Reply:
`initialPriceUpdate` it is has a preset value, which is generated with the pricing data. I don't believe that `time.Now()` would be the correct value in this case.